Lombiq.AuditTrailExtensions.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="Licence.md" Pack="true" PackagePath="" />
+    <None Include="License.md" Pack="true" PackagePath="" />
```

Comment:
You missed one `<PackageLicenseFile>Licence.md</PackageLicenseFile>` on line 18.